README.rst

```diff
@@ -137,9 +137,10 @@ to merge paths from the config file and from the argument:
 
     $ php php-cs-fixer.phar fix --path-mode=intersection /path/to/dir
 
-The ``--format`` option for the output format. Supported formats are ``txt`` (default one), ``json``, ``xml`` and ``junit``.
+The ``--format`` option for the output format. Supported formats are ``txt`` (default one), ``json``, ``xml``, ``checkstyle`` and ``junit``.
 
 NOTE: When using ``junit`` format report generates in accordance with JUnit xml schema from Jenkins (see docs/junit-10.xsd).
+NOTE: When using ``checkstyle`` format report generates in accordance with the "checkstyle" xml schema (see docs/checkstyle.xsd).
```

Comment:
please rephrase, maybe use bullets, no need to have 2 `NOTE:`.

maybe make it linkable, so `Jenkins` is link to `junit-10.xsd` and so ?